openbas-api/src/main/java/io/openbas/rest/user/UserApi.java

```diff
@@ -133,9 +158,18 @@ public ResponseEntity<?> passwordReset(@Valid @RequestBody ResetUserInput input)
     return ResponseEntity.badRequest().build();
```

Comment:
Does that mean this function always return a badrequest?

Reply:
Only if the user is not present (see line 131)

Reply:
... My eyes weren't open sorry. I didn't see the collapsed line on github...